enterprise/coderd/coderd.go

```diff
@@ -674,7 +673,7 @@ func (api *API) Close() error {
 	}
 
 	if api.PrebuildsReconciler != nil {
-		ctx, giveUp := context.WithTimeoutCause(context.Background(), time.Second*30, errors.New("gave up waiting for reconciler to stop"))
+		ctx, giveUp := context.WithTimeoutCause(context.Background(), time.Second*30, xerrors.New("gave up waiting for reconciler to stop"))
```

Comment:
out of scope of this PR: I'd like to see the timeout logic moved inside the prebuilds package; this seems to be the norm for other components.